### PR TITLE
Update use-a-task-sequence-to-manage-virtual-hard-disks.md

### PR DESCRIPTION
**Pull request by sccmentor**
Added in a note to state that the feature has been deprecated since 1710

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

@Article_Author please review (look at the `author` metadata tag)

### CODE REVIEW

**Comment by PRMerger8** [comment]
@sccmentor : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/osd/deploy-use/use-a-task-sequence-to-manage-virtual-hard-disks.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ manager: dougeby
 
 *Applies to: System Center Configuration Manager (Current Branch)*
 
+
+    > [!NOTE] 
+    >  Support for this feature was deprecated in Current Branch version 1710, see [Removed and deprecated features for Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/plan-design/changes/deprecated/removed-and-deprecated-cmfeatures).
```
Comment (aczechowski) [inline sccm/osd/deploy-use/use-a-task-sequence-to-manage-virtual-hard-disks.md]: A few edits here:
- Remove "current branch". I generally just refer to "version yymm". 
- End that first phrase with a period. 
- For the second bit, use the standard formula: "For more information, see (link)."
- Drop the language code ("en-us/") from the URL.

**Comment by PRMerger13** [comment]
@sccmentor : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Looks good @sccmentor, thanks for the revision. :shipit: 
#sign-off
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 19 September) around 11:00 Pacific time.

**Pull request merged**